New package: OpenFST v0.1.1

JuliaRegistrator opened the pull request:
- Registering package: OpenFST
- Repository: https://github.com/laurentbartholdi/OpenFST.jl
- Created by: @laurentbartholdi
- Version: v0.1.1
- Commit: 7a1715cfeb3d0c5211a84ab7b4be577a0744bd2a
- Reviewed by: @laurentbartholdi
- Reference: https://github.com/laurentbartholdi/OpenFST.jl/commit/7a1715cfeb3d0c5211a84ab7b4be577a0744bd2a#commitcomment-144169007
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6312870a25f9246ce93d3913cf4d0cb61a9b545d3a704e390466e7e678b0d0e8886ea6695fe66568d50b2a9efac5c816bfad7f96e7d6a81c1fc2605a308d0866c0516a9d390e9bdd9184eea75f3dfa9e87ab6fb7528e9432e48333ac6acedd7befa96ead43dbd16d966869b3be1dd8f1fe65e15e429e8a81a8451107a673189ff85c9c53bed9cd2658d9f68acd339e0dfec9518548e4c6228462e10c432d83baf4e6d7b2cdf97ccf070ddeaa62890324f -->

laurentbartholdi commented:
[noblock] This name really is meaningful, it's a connection to the "openfst" library.

goerz commented:
The name should be fine. However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README. Also make sure to include a link to the original `openfst` library.

JuliaRegistrator committed:
New package: OpenFST v0.1.1

UUID: 18031d04-943d-420a-99da-58315a5a21aa
Repo: https://github.com/laurentbartholdi/OpenFST.jl.git
Tree: fc6d08341aa93894f70f543cdf4a26314b1b8567

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

laurentbartholdi commented:
> The name should be fine. However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README. Also make sure to include a link to the original `openfst` library.

[noblock] Thanks! I just wrote a short README.md explaining a basic example
